packages/jest-editor-support/src/types.js

```diff
@@ -12,16 +12,22 @@ export type Location = {
   line: number,
 };
 
+export type SpawnOptions = {
+  shell?: boolean,
+};
+
 import type {ChildProcess} from 'child_process';
 import type ProjectWorkspace from './project_workspace';
 
 export type Options = {
   createProcess?: (
     workspace: ProjectWorkspace,
     args: Array<string>,
+    options?: SpawnOptions,
   ) => ChildProcess,
   testNamePattern?: string,
   testFileNamePattern?: string,
+  shell?: boolean,
 };
```

Comment:
@stephtr, you're right. These two type definitions have diverged.
https://github.com/facebook/jest/blob/master/packages/jest-editor-support/index.d.ts#L11